cms/settings/base.py

```diff
@@ -851,4 +852,9 @@
 
 ONS_API_BASE_URL = env.get("ONS_API_BASE_URL")
 
+PUBLISHER_BACKEND = os.getenv("PUBLISHER_BACKEND")
```

Comment:
Let's make the purpose a bit clearer, maybe something like `SEARCH_INDEX_PUBLISHER_BACKEND` or just `SEARCH_INDEX_PUBLISHER`.

Reply:
Okay, made the change! Clearer now!